app/src/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -50,6 +50,7 @@
 import android.widget.Toast;
 
 import org.commcare.android.framework.CommCareActivity;
+import org.commcare.android.framework.SaveSessionCommCareActivity;
 import org.commcare.android.framework.SessionAwareCommCareActivity;
```

Comment:
You should remove the `SessionAwareCommCareActivity`import wherever you use the `SaveSessionCommCareActivity` instead